applications/tari_indexer/src/main.rs

```diff
@@ -56,7 +56,8 @@ async fn main_inner() -> Result<(), ExitError> {
     let cfg = load_configuration(config_path, true, &cli)?;
     let config = ApplicationConfig::load_from(&cfg)?;
     println!("Starting validator node on network {}", config.network);
-
+    // Remove the file if it was left behind by a previous run
+    let _file = fs::remove_file(config.common.base_path.join("pid"));
```

Comment:
Could also be if another instance is already started. Probably fine for our use cases though since we're really just using this for readiness.

Reply:
I suppose we could create a lock and make sure it can't delete it, but if the process previously crashed, it should also delete it